Sources/Sentry/SentryScreenshotIntegration.m

```diff
@@ -10,6 +11,23 @@
 #import "SentrySDK+Private.h"
 
 #if SENTRY_HAS_UIKIT
+
+void
+saveScreenShot(const char *path)
+{
+    NSString *reportPath = [NSString stringWithUTF8String:path];
+    if (![NSFileManager.defaultManager fileExistsAtPath:reportPath isDirectory:nil]) {
```

Comment:
`l`: Why do we pass in `nil`? Can't we just use `[NSFileManager.defaultManager fileExistsAtPath:reportPath]`?

Reply:
We can. I started checking if this was a directory and them changed my mind but keep the method.